### ISSUE TITLE
fails to parse rustc lockfile

### ISSUE THREAD

**Issue by matthiaskrgr** [RustSec/cargo-audit#137]
lockfile: https://github.com/rust-lang/rust/blob/22bc9e1d9ca49ee4f5cd953088ab09c238a6dd26/Cargo.lock

With cargo audit 0.8.1 the file was parsed fine but cargo audit 0.9.1 fails to parse the lockfile:
````
error: Couldn't load Cargo.lock: parse error: parse error: parse error: missing version for dependency: memchr for key `package.dependencies` at line 5022 column 1
````

**Comment by tarcieri** [comment]
Thanks for reporting this. I've fixed the downstream dependency to handle this case: https://github.com/RustSec/cargo-lock/pull/23

**Comment by tarcieri** [comment]
I just released cargo-audit v0.9.2 which should address this. Can you confirm it's fixed for you?

**Comment by matthiaskrgr** [comment]
It works with 0.9.2, thanks!
